mkosi/__init__.py

```diff
@@ -148,13 +148,36 @@ class MkosiException(Exception):
     """Leads to sys.exit"""
 
 
-def print_step(text: str) -> None:
-    sys.stderr.write("‣ \033[0;1;39m" + text + "\033[0m\n")
+class MkosiPrinter():
```

Comment:
If we're doing refactoring anyway, let's take advantage of all the work @behrmann did to allow splitting mkosi up into multiple files and move MkosiPrinter into its own file.

Reply:
moved to `printer.py`